app/src/main/java/org/mozilla/reference/browser/components/Core.kt

```diff
@@ -117,11 +116,16 @@ class Core(private val context: Context) {
         privateMode: Boolean = prefs.getBoolean(context.getPreferenceKey(pref_key_tracking_protection_private), true)
     ): TrackingProtectionPolicy {
 
+        val tackingPolicy = TrackingProtectionPolicy.recommended()
```

Comment:
This should be `trackingPolicy`

Reply:
Let's fix this in a follow-up so we can get you unblocked. :)